test/benchmarks/assembler.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+# pylint: disable=no-member,invalid-name,missing-docstring,no-name-in-module
+# pylint: disable=attribute-defined-outside-init,unsubscriptable-object
+
+from qiskit import assemble
```

Comment:
Depending on how far back we want to go on this it might make more sense to change this import to:

```suggestion
from qiskit.compiler import assemble
```

Since we added the top level import more recently than when the assemble function was added.